monitoring/setup.py

```diff
@@ -33,6 +33,7 @@
     'google-api-core<2.0.0dev,>=0.1.1',
 ]
 extras = {
+    'optional': ['pandas>=0.22.0'],
```

Comment:
we've generally named our extras after the thing they pull in, so I would prefer this extra be called "pandas"

Reply:
Changed (and reflected on the README).